README.md

```diff
@@ -0,0 +1,6 @@
+# cmus Flatpak
+
+[Icon](https://github.com/cmus/cmus/issues/965) and [desktop file](https://github.com/cmus/cmus/issues/978) not from upstream.
+
+## Current issues
+- MPRIS fails with "connection reset by peer" without full session bus permissions
```

Comment:
At first glance, other apps do this: `--own-name="org.mpris.MediaPlayer2.firefox.*"`. But I assume there's a different problem with the name cmus uses.

Reply:
I tried adding the * in, as well as the quotes, neither fixed it. There's more details on the problem in flatpak/flatpak#3978, looks like it's a disagreement between sd-bus (the systemd dbus implementation, used by cmus) and Flatpak's xdg-dbus-proxy.